test/workbox-webpack-plugin/node/inject-manifest.js

```diff
@@ -240,12 +256,87 @@ describe(`[workbox-webpack-plugin] InjectManifest (End to End)`, function() {
 
         const swFile = path.join(outputDir, path.basename(SW_SRC));
         try {
+          // First, validate that the generated service-worker.js meets some basic assumptions.
+          await validateServiceWorkerRuntime({swFile, expectedMethodCalls: {
+              importScripts: [[
+                FILE_MANIFEST_NAME,
+                workboxEntryName,
+              ]],
+              suppressWarnings: [[]],
+              precacheAndRoute: [[[], {}]],
+            }});
+
+          // Next, test the generated manifest to ensure that it contains
+          // exactly the entries that we expect.
+          const manifestFileContents = await fse.readFile(path.join(outputDir, FILE_MANIFEST_NAME), 'utf-8');
+          const context = {self: {}};
+          vm.runInNewContext(manifestFileContents, context);
+
+          const expectedEntries = [{
+            url: 'entry2-0c3c00f8cd0d3271089c.js',
+          }, {
+            url: 'entry1-3865b3908d1988da1758.js',
+          }];
+          expect(context.self.__precacheManifest).to.eql(expectedEntries);
+
+          done();
+        } catch (error) {
+          done(error);
+        }
+      });
+    });
+
+    it(`should support setting importWorkboxFrom to 'local'`, function(done) {
```

Comment:
It'd be cool to include a test where Webpack `publicPath` is specified

Reply:
Done.